config/compiler/kind_registry.yaml

```diff
@@ -69,6 +69,16 @@ passes:
             kind: composable_slot
             composable_slot: { variant: dataquery }
 
+  # typed as `links?: [...]` in the original schema
```

Comment:
Would there be any needed change here, when upstream original schema gets augmented from current _any_ type for `links`?

Reply:
We'd need to remove this change, or it would override the updated upstream schema :)

But since dashboard-related schemas are being revamped, there will likely never be a fix for this. Instead, we'll move over to the "dashboard v2" schemas.